namespaces/cloud-platform-live-0.k8s.integration.dsd.io/laa-apply-for-legalaid-uat/resources/main.tf

```diff
@@ -0,0 +1,54 @@
+terraform {
+  backend "s3" {}
+}
+
+provider "aws" {
+  region = "eu-west-1"
+}
+
+/*
+ * When using this module through the cloud-platform-environments, the following
+ * two variables are automatically supplied by the pipeline.
+ *
+ */
+
+variable "cluster_name" {}
+
+variable "cluster_state_bucket" {}
+
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+module "apply-for-legal-aid-rds" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=2.3"
+
+  cluster_name           = "${var.cluster_name}"
+  cluster_state_bucket   = "${var.cluster_state_bucket}"
+  team_name              = "apply-for-legal-aid"
+  business-unit          = "laa"
+  application            = "laa-apply-for-legal-aid"
+  is-production          = "false"
+  environment-name       = "uat"
+  infrastructure-support = "apply@digtal.justice.gov.uk"
+  db_engine              = "postgres"
+  db_engine_version      = "10.5"
+  db_name                = "apply-for-legal-aid"
```

Comment:
I'm not sure whether this is a valid name for a database in Postgres (the dashes). See under `DBName` [here](https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_CreateDBInstance.html). Other sources online seem to indicate that it's possible but creates various issues because it needs quoting. AWS RDS might have put an extra limitation on it.

You can leave it as-is and adjust if it fails to apply if you don't want to change it now.

(also, if you omit the value it will auto-generate a random db name)

Reply:
I've amended to "apply_for_legal_aid"